app/models/validator_history.rb

```diff
@@ -55,6 +55,17 @@ class ValidatorHistory < ApplicationRecord
     )
   end
 
+  scope :validator_histories_from_period, ->(network:, account:, from:, to:, limit:) do
+    where(
+      "network = ? AND account = ? AND created_at BETWEEN ? AND ?",
```

Comment:
```suggestion
      network: network, account: account, created_at: from...to,
```
Give it a try, I think it should generate the same SQL and looks nicer